app/main/views/services.py

```diff
@@ -51,14 +51,28 @@ def view(service_id):
         flash({'api_error': service_id}, 'error')
         return redirect(url_for('.index'))
 
+    most_recent_audit_events = data_api_client.find_audit_events(
+        latest_first="true",
+        object_id=service_id,
+        object_type="services",
+        audit_type=AuditTypes.update_service_status
+    )
+    if most_recent_audit_events.get('auditEvents'):
+        last_updated_by = most_recent_audit_events['auditEvents'][0]['user']
```

Comment:
I think `removed_by` and `removed_on` would be better variable names here - the service might have been updated since it was removed, but it's the removal we care about here.